Libplanet.Common/HashDigest.cs

```diff
@@ -1,3 +1,6 @@
+#if NETSTANDARD2_0
+using System.Runtime.CompilerServices;
+#endif
```

Comment:
Is it okay not to be alphabetically sorted?

Reply:
This is automatically sorted by rider. I'll get it back previous position.